docs/intro.md

```diff
@@ -20,7 +20,7 @@ If you're interested in a more in-depth treatment of the HLS format, check out [
 ## Dynamic Adaptive Streaming over HTTP
 Similar to HLS, [DASH][dash-wiki] content is segmented and is delivered over HTTP(s).
 
-A DASH stream consits of a *Media Presentation Description*(MPD) that describes segment metadata. Each segment must contain either ISO base media file format(e.g MP4) or MPEG-2 TS data. Typically the MPD will describe the various *Representations* that map to collections of segments at different bitrates to allow bitrate selection.
+A DASH stream consits of a *Media Presentation Description*(MPD) that describes segment metadata and includes links to the segments themselves. Each segment can contain either ISO base media file format(e.g MP4) or MPEG-2 TS data. Typically the MPD will describe the various *Representations* that map to collections of segments at different bitrates to allow bitrate selection.
```

Comment:
It might be easiest to go with something like the wikipedia description: "A media presentation description (MPD) describes segment information (timing, URL, media characteristics like video resolution and bit rates), and can be organized in different ways such as SegmentList, SegmentTemplate, SegmentBase and SegmentTimeline, depending on the use case."